tests/console/suseconnect_scc.pm

```diff
@@ -39,7 +39,7 @@ sub run {
 
 
     $self->select_serial_terminal;
-    die 'SUSEConnect package is not pre-installed!' if script_run 'rpm -q SUSEConnect';
+    die 'SUSEConnect package is not pre-installed!' if script_run 'SUSEConnect -s';
```

Comment:
If you just want to check if an application is present without relying on the package name then better use `command -v SUSEConnect`

```suggestion
    die 'SUSEConnect package is not pre-installed!' if script_run 'command -v SUSEConnect';
```

Reply:
That makes perfect sense, thanks for the suggestion.